training/src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,124 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false, unique = true)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private int pages;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    public Book() { }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
Knowing that we'll be using the autogenerated id, this setId method is not necessary. This way we can avoid someone change the id through our code

Reply:
fixed